stdlib/experimental/geo/README.md

```diff
@@ -254,14 +292,231 @@ asTracks = (tables=<-, groupBy=["id","tid"], orderBy=["_time"]) =>
+
+Input parameters are:
+- `region`
+- `geometry`
+
+Example:
+```js
+box = {minLat: 40.51757813, maxLat: 40.86914063, minLon: -73.65234375, maxLon: -72.94921875}
+
+from(bucket:"mta")
+    ...
+    |> geo.toRows()
+    |> map(fn: (r) => ({
+      r with st_contains: ST_Contains(region: region, geometry: {lat: r.lat, lon: r.lon})
+    }))
+```
+
+```js
+box = {minLat: 40.51757813, maxLat: 40.86914063, minLon: -73.65234375, maxLon: -72.94921875}
+
+from(bucket:"mta")
+    ...
+    |> geo.toRows()
+    |> geo.asTracks()
+    |> geo.ST_LineString()
+    |> map(fn: (r) => ({
+      r with st_contains: ST_Contains(region: region, geometry: {linestring: r.st_linestring})
+    }))
+```
+
+### Function `ST_Distance`
+
+Returns distance between specified region and geometry.
+Parameter `geometry` can be either a point or a linestring.
+
+Input parameters are:
+- `region`
+- `geometry`
+
+Example:
+```js
+box = {minLat: 40.51757813, maxLat: 40.86914063, minLon: -73.65234375, maxLon: -72.94921875}
+
+from(bucket:"mta")
+    ...
+    |> geo.toRows()
+    |> map(fn: (r) => ({
+      r with st_distance: ST_Distance(region: region, geometry: {lat: r.lat, lon: r.lon})
+    }))
+```
+
+### Function `ST_DWithin`
+
+Returns boolean if geometry is within a distance to specified region.
+Parameter `geometry` can be either a point or a linestring.
+
+Input parameters are:
+- `region`
+- `geometry`
+
+Example:
+```js
+box = {minLat: 40.51757813, maxLat: 40.86914063, minLon: -73.65234375, maxLon: -72.94921875}
+
+from(bucket:"mta")
+    ...
+    |> geo.toRows()
+    |> map(fn: (r) => ({
+      r with st_within: ST_DWithin(region: region, geometry: {lat: r.lat, lon: r.lon}, distance: 15.0)
+    }))
+```
+
+#### Function definition
+
+```js
+ST_DWithin = (region, distance, geometry) =>
+  ST_Distance(region: region, geometry: geometry) <= distance
+```
+
+### Function `ST_Intersects`
+
+Returns boolean whether geometry intersects specified region.
+Parameter `geometry` can be either a point or a linestring.
+
+Example:
+```js
+box = {minLat: 40.51757813, maxLat: 40.86914063, minLon: -73.65234375, maxLon: -72.94921875}
+
+from(bucket:"mta")
+    ...
+    |> geo.toRows()
+    |> geo.asTracks()
+    |> geo.ST_LineString()
+    |> map(fn: (r) => ({
+      r with st_intersects: ST_Intersects(region: region, geometry: {linestring: r.st_linestring})
+    }))
+```
+
+#### Function definition
+
+```js
+ST_Intersects = (region, geometry) =>
+  ST_Contains(region: region, geometry: geometry)
+```
+
+### Function `ST_Length`
+
+Returns spherical length of specified geometry.
+Parameter `geometry` can be either a point (result is 0.0) or a linestring.
+
+Example:
+```js
+from(bucket:"mta")
+    ...
+    |> geo.toRows()
+    |> geo.asTracks()
+    |> geo.ST_LineString()
+    |> map(fn: (r) => ({
+      r with st_length: ST_Length(geometry: {linestring: r.st_linestring})
+    }))
+```
+
+### Function `ST_LineString`
+
+_This is a proprietary function._
```

Comment:
Is this something that some other vendors support but that is not standard, or something totally specific to Flux?

Reply:
Totally specific to Flux.

It "converts" a series containing points with _lat_,_lon_ to a string formatted as LINESTRING which is a GIS type for representing simple track.

Reply:
It is wrong description, such function is actually quite common.